qemu/tests/rh_kselftests_vm.py

```diff
@@ -41,14 +41,15 @@ def run(test, params, env):
         s, o = session.cmd_status_output(tests_execution_cmd, 180)
         test.log.info("The selftests results: %s", o)
 
-        summary = re.findall(r"\# SUMMARY.+", o)
-        num_failed_tests = int(re.findall(r"FAIL\=\d+", str(summary))[0].split("=")[1])
+        fail_match = re.search(r"SUMMARY:.+FAIL=(?P<fail>\d+?)", o)
```

Comment:
what about using one search for all, e.g. 
m = re.search(r"SUMMARY:.+FAIL=(?P<fail>\d+ SKIP=(?P<skip>\d+))", s), o)
m.group("fail")
m.group("skip")

Reply:
@zhencliu updated to use a single search !